marshmallow/fields.py

```diff
@@ -1172,14 +1172,15 @@ def _deserialize(self, value, attr, data, **kwargs):
             self.fail('invalid')
 
 
-class Dict(Field):
-    """A dict field. Supports dicts and dict-like objects. Optionally composed
+class Map(Field):
```

Comment:
The typing and collections modules call this abstract class `Mapping`.

https://docs.python.org/3/library/typing.html#typing.Mapping

https://docs.python.org/3/library/collections.abc.html#collections.abc.Mapping

Reply:
Let's call this field Mapping and make alias Map = Mapping?